apps/microsoft-teams/app-actions/src/actions/send-test.ts

```diff
@@ -35,20 +31,17 @@ export const handler = withAsyncAppActionErrorHandling(
       contentTypeName,
     };
 
-    response = await helpers.sendTestMessage(
-      config.botServiceUrl,
-      config.apiKey,
-      tenantId,
-      testMessagePayload
+    const msTeamsBotServiceResponse = await config.msTeamsBotService.sendTestMessage(
+      testMessagePayload,
+      tenantId
     );
 
-    if (!response.ok) {
-      throw new Error(response.error ?? 'Failed to send test message');
+    // if we align the error types from the bot service and our app action error we could just return
```

Comment:
Yes! When we update the bot service error to be an object, not a string, then we can do this.